packages/datetime/src/common/dateUtils.ts

```diff
@@ -110,6 +110,55 @@ export function isMonthInRange(date: Date, dateRange: DateRange) {
     return start <= day && day <= end;
 }
 
+export function isTimeInRange(date: Date, minDate: Date, maxDate: Date): boolean {
+    const time = getDateOnlyWithTime(date).getTime();
+    const minTime = getDateOnlyWithTime(minDate).getTime();
+    const maxTime = getDateOnlyWithTime(maxDate).getTime();
+
+    const isTimeGreaterThanMinTime = time >= minTime;
+    const isTimeSmallerThanMaxTime = time <= maxTime;
+
+    if (maxTime <= minTime) {
+        return isTimeGreaterThanMinTime || isTimeSmallerThanMaxTime;
+    }
+
+    return isTimeGreaterThanMinTime && isTimeSmallerThanMaxTime;
+}
+
+export function getTimeInRange(time: Date, minTime: Date, maxTime: Date) {
+    if (timesAreEqual(minTime, maxTime)) {
```

Comment:
I'm not sure if this check is a responsibility of this function, what do you think?